java/src/jmri/jmrix/loconet/LnOpsModeProgrammer.java

```diff
@@ -41,6 +41,7 @@
  * @author Bob Jacobsen Copyright (C) 2002
  * @author B. Milhaupt, Copyright (C) 2018
  * @author Egbert Broerse, Copyright (C) 2020
+ * @author Andrew Crosland, Copyright (C) 2021
```

Comment:
It would be good to remove this comment now when the new `readCV` method is removed from this class.